src/main/java/edu/harvard/iq/dataverse/api/ApiConfiguration.java

```diff
@@ -1,8 +1,18 @@
 package edu.harvard.iq.dataverse.api;
 
 import javax.ws.rs.ApplicationPath;
-import javax.ws.rs.core.Application;
+import org.glassfish.jersey.media.multipart.MultiPartFeature;
+import org.glassfish.jersey.server.ResourceConfig;
```

Comment:
@scolapasta are we consciously switching from a JAVA EE standard (JAX-RS) to an specific implementation (Jersey)?